src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,101 @@
+        for (int i = 0; i < allStrings.length; i = i + 2) {//[0]buy
+            if(allStrings[i].equals(BUY)) {
+                buyScore += Integer.parseInt(allStrings[i + 1]);
+                stringBuilderBuy.append(allStrings[i + 1]).append(',');
+                countBuyLength++;
+            }
+            if(allStrings[i].equals(SUPPLY)) {//+
+                supplyScore += Integer.parseInt(allStrings[i + 1]);
+                stringBuilderSupply.append(allStrings[i + 1]).append(",");
+                countSupplyLength++;
+            }
+        }
+        String[] buyWithComma = stringBuilderBuy.toString().split(",");
+        String[] supplyWithComma = stringBuilderSupply.toString().split(",");
+
+        String[] buy = new String[countBuyLength];
+        String[] supply  = new String[countSupplyLength];
+        int counterOfBuy = 0;
+        int counterOfSupply = 0;
+        for (String current : buyWithComma) {
+            if (!current.equals(",")) {
+                buy[counterOfBuy] = current;
+                counterOfBuy++;
+            }
+        }
+        for (String current : supplyWithComma) {
+            if (!current.equals(",")) {
+                supply[counterOfSupply] = current;
+                counterOfSupply++;
+            }
+        }
+        stringBuilderBuy.setLength(0);
+        stringBuilderBuy.append("buy = ");
+        stringBuilderBuy.setLength(0);
+        for (int i = 0; i < buy.length; i++) {
+            stringBuilderBuy.append(buy[i]);
+            if(i != buy.length - 1) {
+                stringBuilderBuy.append(" + ");
+            }
+            if (i == buy.length - 1) {
+                stringBuilderBuy.append(" = ").append(buyScore);
+            }
+        }
+        String resultBuy = stringBuilderBuy.toString();
+        stringBuilderSupply.setLength(0);
+        stringBuilderSupply.append("supply = ");
+        stringBuilderSupply.setLength(0);
+        for (int i = 0; i < supply.length; i++) {
+            stringBuilderSupply.append(supply[i]);
+            if(i != supply.length - 1) {
+                stringBuilderSupply.append(" + ");
+            }
+            if (i == supply.length - 1) {
+                stringBuilderSupply.append(" = ").append(supplyScore);
+            }
+        }
+        String resultSupply = stringBuilderSupply.toString();
+        String resultFinal = stringBuilderResult.append("result = supply - buy = ")
+                .append(supplyScore).append(" - ").append(buyScore).append(" = ")
+                .append(supplyScore - buyScore).toString();
 
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(fileToWrite))) {
+            writer.write(resultSupply + System.lineSeparator());
+            writer.write(resultBuy + System.lineSeparator());
+            writer.write(resultFinal + System.lineSeparator());
+        } catch (IOException e) {
+            throw new RuntimeException("Cannot write in file",e);
+        }
     }
```

Comment:
enormous big method, doing more action rather 1 - violates SRP. refactor, using this template
```suggestion
        String dataFromFile = readFromFile(fromFileName);
        String report = processData(dataFromFile);
        writeToFile(report, toFileName);
```

Reply:
Sorry! i did pull request but it not finall wersion, appologaze..

Reply:
Sorry! i did pull request but it not final version, apologise..

Reply:
Now its ready :)